configuration/pih/htmlforms/section-delivery.xml

```diff
@@ -826,184 +776,6 @@
                 </div>
             </div>
         </div>
-       
```

Comment:
@Dutervil Did you want to remove the postpartum section?

Reply:
No, I did not